cmd/root.go

```diff
@@ -278,15 +278,7 @@ func (h *Hammer) installImage(eventEmitter *event.EventEmitter, m *models.Models
 
 	err = rep.ReportInstallation()
 	if err != nil {
-		wait := 10 * time.Second
```

Comment:
is still a reboot triggered after this ?

Reply:
[Yes](https://github.com/metal-stack/metal-hammer/blob/7d71dbb5bbb8d2bac78e3b3ebd95af3a0e1b9ed7/main.go#L74) and finally [here](https://github.com/metal-stack/metal-hammer/blob/7d71dbb5bbb8d2bac78e3b3ebd95af3a0e1b9ed7/main.go#L82)